config/webhook/manifests.yaml

```diff
@@ -449,19 +449,18 @@ webhooks:
     service:
       name: webhook-service
       namespace: system
-      path: /validate-infrastructure-cluster-x-k8s-io-v1beta1-azuremachinepoolmachine
+      path: /validate-infrastructure-cluster-x-k8s-io-v1alpha1-azureasomanagedcluster
```

Comment:
Something about the new API versions is causing these to sort differently, but AFAICT the elements themselves aren't changed aside from the intended API version-related tweaks.